src/toolbox.ts

```diff
@@ -30,77 +31,84 @@ enum ValidResizeValues {
+ */
+String.prototype.replaceLowerConcat = function (
+    before: string,
+    after: string,
+    concat_string: string = null,
+) {
+    if (typeof concat_string === "string") {
+        return this.replaceAll(before, after)
+            .toLowerCase()
+            .concat(concat_string);
     }
 
-    return this.replaceAll(before, after).toLowerCase()
-}
-
+    return this.replaceAll(before, after).toLowerCase();
+};
 
 /**
  * Manager for toolbox. Contains ToolboxTab items.
  */
 export class Toolbox {
-    
     constructor(
         public tabs: ToolboxTab[] = [],
-        public items: ToolboxItem[] = []
-    ) {
-    }
+        public items: ToolboxItem[] = [],
+    ) {}
 
-    public static create_toolbox(ulabel: ULabel, toolbox_item_order: unknown[]) {
+    public static create_toolbox(
+        ulabel: ULabel,
+        toolbox_item_order: unknown[],
+    ) {
         // Grab the default toolbox if one wasn't provided
         if (toolbox_item_order == null) {
-            toolbox_item_order = ulabel.config.toolbox_order
+            toolbox_item_order = ulabel.config.toolbox_order;
         }
 
         // There's no point to having an empty toolbox, so throw an error if the toolbox is empty.
         // The toolbox won't actually break if there aren't any items in the toolbox, so this
         // error isn't strictly neccesary.
         if (toolbox_item_order.length === 0) {
-            throw new Error("No Toolbox Items Given")
+            throw new Error("No Toolbox Items Given");
         }
 
-        this.add_styles()
+        this.add_styles();
 
-        let toolbox_instance_list = [];
+        const toolbox_instance_list = [];
         // Go through the items in toolbox_item_order and add their instance to the toolbox instance list
         for (let i = 0; i < toolbox_item_order.length; i++) {
-
             let args: object, toolbox_key: number;
 
-            // If the value of toolbox_item_order[i] is a number then that means the it is one of the 
+            // If the value of toolbox_item_order[i] is a number then that means the it is one of the
             // enumerated toolbox items, so set it to the key, otherwise the element must be an array
             // of which the first element of that array must be the enumerated value, and the arguments
             // must be the second value
-            if (typeof(toolbox_item_order[i]) === "number") {
-                toolbox_key = <number> toolbox_item_order[i]
+            if (typeof toolbox_item_order[i] === "number") {
+                toolbox_key = <number>toolbox_item_order[i];
             } else {
                 toolbox_key = toolbox_item_order[i][0];
-                args = toolbox_item_order[i][1]  
+                args = toolbox_item_order[i][1];
             }
 
-            let toolbox_item_class = ulabel.config.toolbox_map.get(toolbox_key);
+            const toolbox_item_class =
+                ulabel.config.toolbox_map.get(toolbox_key);
```

Comment:
82 character line ;_; 

Given this one is a loop, probably fine to store `ulabel.config.toolbox_map` to a `const` anyways. 

My other comment on line length is [here](https://github.com/SenteraLLC/ulabel/pull/175/files#r1781120529).
@TrevorBurgoyne @csolbs24 assuming we come to some agreement on line length, should I go ahead and clean up cases like this for readability as part of this PR, or should I open an issue so we can address those separately?

Additionally, if we're making a decision for those cases that isn't enforceable by the formatter, we should probably add it to documentation somewhere as a recommendation. We absolutely don't need our own fully-fledged style guide, so if we're concerned about it snowballing to that, I'm fine skipping on that.